modules/managed-cluster-notification-types.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-notifications.adoc
+// * osd_cluster_admin/osd-cluster-notifications.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="managed-cluster-notification-types_{context}"]
+= Cluster notification types
+
+Each cluster notification has an associated notification type to help you identify notifications that are relevant to your role and responsibilities. You can filter cluster notifications according to these types in the {hybrid-console}, in the **Cluster history** tab for your cluster.
+
+Red Hat uses the following notification types to indicate notification relevance.
+
+Capacity management:: Notifications for events related to updating, creating, or deleting node pools, machine pools, compute replicas or quotas (load balancer, storage, etc.).
+Cluster access:: Notifications for events related to adding or deleting groups, roles or identity providers, for example, when SRE cannot access your cluster because STS credentials have expired, when there is a configuration problem with your AWS roles, or when you add or remove identity providers.
+Cluster add-ons:: Notifications for events related to add-on management or upgrade maintenance for add-ons, for example, when an add-on is installed, upgraded, or removed, or cannot be installed due to unmet requirements.
+Cluster configuration:: Notifications for cluster tuning events, workload monitoring, and inflight checks.
+Cluster lifecycle:: Notifications for cluster or cluster resource creation, deletion, and registration, or change in cluster or resource status (for example, ready or hibernating).
+Cluster networking:: Notifications related to cluster networking, including HTTP/S proxy, router, and ingress state.
+Cluster ownership:: Notifications related to cluster ownership transfer from one user to another.
+Cluster scaling:: Notifications related to updating, creating, or deleting node pools, machine pools, compute replicas or quota.
+Cluster security:: Events related to cluster security, for example, an increased number of failed access attempts, updates to trust bundles, or software updates with security impact.
+Cluster subscription:: Cluster expiration, trial cluster notifications, or switching from free to paid.
+Cluster updates:: Anything relating to upgrades, such as upgrade maintenance or enablement.
+Customer support:: Updates on support case status.
+// Omitted, as no definition provided as part of OSDOCS-8938
```

Comment:
@laubai , Did we also omit the "General notification" type? I Just found that it is missing here.
![image](https://github.com/openshift/openshift-docs/assets/93653094/bfef1ce1-3298-412a-92da-8382d5c0aab7)

Reply:
I omitted this from the document on the advice of PM, as ideally all notifications will have a more specific type, but can add something back in about this.